modules/nodes-nodes-tls-configuring.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-nodes-tls.adoc
+
+[id="nodes-nodes-tls-configuring_{context}"]
+= Configuring TLS profiles for the Kubelet
+
+To configure a TLS security profile for the kubelet HTTP server, create a `KubeletConfig` object to specify a TLS profile type or create a custom profile, and node match labels.
+
+.Procedure
+
+. Create a `KubeletConfig` custom resource (CR) to configure the TLS security profile type:
++
+.Sample `KubeletConfig` CR for a `Custom` profile
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: set-kubelet-tls-security-profile
+spec:
+  audit:
```

Comment:
audit can be removed... Where is this coming from?

Reply:
@rphillips It came from an earlier test case, long since edited. I will remove it here. Thanks!